app/views/index_of_poets/items/browse_facet.html.erb

```diff
@@ -30,14 +30,13 @@
     <div role="tabpanel"
       class="tab-pane <%= active?(facet_name, @browse_facet) %>" id="<%= id %>">
       <% if @res.key?(facet_name) %>
-        <%# switch for person.name? %>
-        <% if @browse_facet.include?"person.name" %>
-          <div class="btn-group">
-          <%= button_to "A-G", index_of_poets_browse_facet_path("person.name_ag"), method: :get, :class => "btn btn-default" %>
-          <%= button_to "H-M", index_of_poets_browse_facet_path("person.name_hm"), method: :get, :class => "btn btn-default" %>
-          <%= button_to "N-Z", index_of_poets_browse_facet_path("person.name_nz"), method: :get, :class => "btn btn-default" %>
+        <%# switch for title %>
+        <% if @browse_facet.include?"title" %>
+          <%= button_to "A-G", index_of_poets_browse_facet_path("title_ag"), method: :get %>
```

Comment:
The following 3 lines should have `:class => "btn btn-default"` - I don't know why main overwrote this change from the `accessibility_sprint` > `dev` PR, but didn't overwrite any of the other changes from that PR.